### DISCUSSION TITLE
Import/export functions across blogs

### DISCUSSION THREAD

**Discussion by tiny-wesker** [AprilSylph/XKit-Rewritten#723] [Suggestion Box]
Batch block isn't maintained anymore, but it was handy to be able to back up externally and/or transfer to a new blog.
Is it feasible to implement something like that, exporting blocks, follows, etc to the extension or a text file?
Something similar I've had in mind but would probably be more involved, would be exporting drafts or likes similar to how tumblr's blog backup function works, since native tumblr will only save your posts.

**Comment by Tarakore** [comment]
Was about to make a suggestion for this exact thing. I use a backup of the Batch Block addon that's a little questionably maintained but it's just about the only option I have for moving blocklists around. I'd kill for even just the importing feature alone.
